beta/iam/setup-sdk.md

```diff
@@ -39,6 +39,12 @@ implementation('com.adobe.marketing.mobile:messaging:1.2.0-beta-1-SNAPSHOT')
 implementation('com.adobe.marketing.mobile:optimize:1.0.0-SNAPSHOT')
 ```
 
+If you use the Messaging IAM beta extension alongside the Campaign Standard extension, the latest Campaign Standard extension must be used to resolve a compatibility issue:
```

Comment:
We should specify that we need Campaign Standard extension >= 1.0.9 as latest will change with a couple more releases. I suppose we can specify a [version range](https://docs.gradle.org/current/userguide/single_versions.html) with gradle `[1.0.9,)` with an inclusive lower bound

Reply:
Also should we specify - `If you use the Messaging extension (In-App beta) alongside...`